lang/perl/lib/Avro/Schema.pm

```diff
@@ -284,6 +284,7 @@ sub is_data_valid {
     }
     if ($type eq 'long') {
         if ($Config{use64bitint}) {
+            no warnings;
```

Comment:
Wouldn't be better to check whether `$data` is defined/initialized ?

Reply:
Good point. I had the same idea but ended up deciding in favor of following the same approach in "int" validation (just a couple of lines above), which seems to have been stable for a long time.